pkg/cmd/server/start/start_master.go

```diff
@@ -648,8 +673,11 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 	}
 
 	controllerContext := kctrlmgr.ControllerContext{
-		ClientBuilder:      saClientBuilder,
-		InformerFactory:    oc.Informers.KubernetesInformers(),
+		ClientBuilder: saClientBuilder,
+		InformerFactory: genericInformers{
+			SharedInformerFactory: oc.Informers.KubernetesInformers(),
+			generic:               []GenericResourceInformer{oc.Informers},
```

Comment:
Isn't this wiring in the hand coded informers?  I'm against including them in new places.